ebill_paynet/models/paynet_service.py

```diff
@@ -62,13 +66,20 @@ def _compute_url(self):
             else:
                 record.url = SYSTEM_PROD_URL
 
+    def _get_dws_connection(self):
+        return PayNetDWS(
+            self.url,
+            self.use_test_service,
+            self.operation_timeout or None
```

Comment:
```suggestion
            operation_timeout=self.operation_timeout or None
```
This is a named argument.

Reply:
This code already works like this too. What makes a keyword/named argument is the usage, not the declaration of the function/method. All parameters here could be named arguments, or positional ones (like in this case).
```python
PayNetDWS(operation_timetout=10, test_service='test', url='test')    # named arguments
# same than
PayNetDWS('test', 'test', 10)    # positional arguments
```

Reply:
Yes I know, that's why I approved the PR anyway.